### ISSUE TITLE
Remove the slf4j-nop from compile classpath

### ISSUE THREAD

**Issue by sureshg** [Password4j/password4j#46]
You shouldn't add the `slf4j` `nop` impl to the classpath ( https://github.com/Password4j/password4j/blob/master/pom.xml#L53) as it causing package conflict issues for modular applications. Usually application using this library brings proper slf4j impl.

```
 error: module dev.xxx.openjdk.playground reads package org.slf4j.impl from both org.slf4j.nop and org.apache.logging.log4j.slf4j
```

**Comment by firaja** [comment]
Merged into master

**Comment by firaja** [comment]
Hello @pnacelli and @sureshg,
I released 1.5.4 to the maven repo with this issue fixed.

**Comment by pnacelli** [comment]
Hi @firaja got your notification, much appreciated! Waiting for the Maven repo to index your push
![image](https://user-images.githubusercontent.com/689539/142645607-12f5c2ad-a169-42b9-bd09-3582f37c7047.png)
Have a great weekend! 🙂
